src/script/event/WebSocketService.js

```diff
@@ -215,30 +214,16 @@ z.event.WebSocketService = class WebSocketService {
   sendPing() {
     const isReadyStateOpen = this.socket.readyState === 1;
     if (isReadyStateOpen) {
-      if (this._pingHasExperiencedSuspiciousInactivity()) {
+      if (this.hasAlreadySentUnansweredPing) {
         this.logger.warn('Ping interval check failed');
         return this.reconnect(WebSocketService.CHANGE_TRIGGER.PING_INTERVAL);
```

Comment:
Done @bennyn, this is now based on a counter (used boolean for readability), a second ping in the stack now causes reconnect as you can see right in this snippet

Reply:
Quick question that I cannot seem to answer just looking at the code. 
Will the non-time-based solution work in case of the laptop being put in sleep mode?

Previously the time would also tell us if the computer is not active (and thus not sending any ping). This was why time is an important component here. 
I believe with this solution, if the computer is put to sleep we will not reconnect automatically. 

Now this scenario might actually be covered by the fact that we are going to send pings that are not going to be replied to. Can you confirm this @maximbaz ?

Reply:
I've just confirmed that both putting computer to sleep and simply shutting down wifi doesn't affect Wire, when Internet is back Wire does reconnect as expected. However I can also see that Wire is detecting "connection lost" and "connection regained", so when connection is lost, WebsocketService is reset and no pings are being sent. Once "connection regained" occurs, pings are being sent again as expected.

Reply:
Awesome!! This is a great PR @maximbaz Thanks a lot :+1: